demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
+1. Download the english language file.
+1. Add the translation keys in [en_translations](en_translations.properties)
+1. Upload the translation file. 
+1. Upload the contact forms using medic-conf `medic-conf upload-contact-forms`
+1. Now you will have a few custom contact types added to your instance. 
```

Comment:
Maybe add another step to tell the user where to see these  or how to verify the new contact types have been added?

Reply:
Will do.